config/jobs/kubernetes-sigs/node-feature-discovery-operator/node-feature-discovery-operator-presubmits.yaml

```diff
@@ -0,0 +1,33 @@
+presubmits:
+  kubernetes-sigs/node-feature-discovery-operator:
+  - name: pull-node-feature-discovery-operator-verify
+    always_run: true
+    decorate: true
+    annotations:
+      testgrid-dashboards: sig-node-node-feature-operator-discovery
```

Comment:
This should be `sig-node-node-feature-discovery`

Reply:
woops.....